src/usr/share/gpgmailer/gpgmailer.py

```diff
@@ -14,19 +14,24 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 __all__ = ['GpgMailer']
-__author__ = 'Joel Luellwitz, Emily Frost, and Brittney Scaccia'
```

Comment:
You should add Brittney back (unless she has no commits). This isn't a copyright. It is credit.